db/schema.rb

```diff
@@ -1290,7 +1290,7 @@
   add_foreign_key "spree_roles_users", "spree_roles", column: "role_id", name: "spree_roles_users_role_id_fk"
   add_foreign_key "spree_roles_users", "spree_users", column: "user_id", name: "spree_roles_users_user_id_fk"
   add_foreign_key "spree_shipments", "spree_addresses", column: "address_id", name: "spree_shipments_address_id_fk"
-  add_foreign_key "spree_shipments", "spree_orders", column: "order_id", on_delete: :cascade
+  add_foreign_key "spree_shipments", "spree_orders", column: "order_id", name: "spree_shipments_order_id_fk", on_delete: :cascade
```

Comment:
:thinking: it's weird that suddenly ActiveRecord gives these foreign keys a name... I guess it's fine. We won't load the schema in production.

Reply:
I've been trying to diagnose various dev-environment issues this week (including issues around running migrations), and I think they were caused by the dual-boot setup...

Reply:
ah! that's very likely

Reply:
Sometimes I get weird things going on unless I comment out all the dualboot code... :see_no_evil:

Reply:
to avoid that I suggest we leave any changes on the db/schema.rb out of the commits dealing with Rails 5. When the time comes to move to it, we can run `rake db:schema:dump`. That would mean we can't rely on `rake db:schema:load` for local development on Rails 5, but I guess that's fine @luisramos0 .